Source/Core/VideoCommon/ConstantManager.h

```diff
@@ -8,6 +8,7 @@
 
 // all constant buffer attributes must be 16 bytes aligned, so this are the only allowed components:
 typedef float float4[4];
+typedef float float2[2];
```

Comment:
We can't use a float2 type, as it's not 16 byte aligned.

Instead use a float4 and leave the last two entries empty. Or you might get lucky and find one of the other float4s already has two empty entries.